collections/datasets/datapublisher.php

```diff
@@ -302,6 +302,9 @@ function checkAllColl(cb) {
 							<input type="hidden" name="colliddel" value="<?php echo $dArr['collid']; ?>">
 							<input type="hidden" name="collid" value="<?php echo $dArr['collid']; ?>">
 							<input type="image" src="../../images/del.png" name="action" alt="<?php echo $LANG['DELETE_ARCHIVE']; ?>" value="DeleteCollid" title="<?php echo $LANG['DELETE_ARCHIVE']; ?>" style="width:1.2em;">
+						<button type="image" src="../../images/del.png" name="action" alt="<?php echo $LANG['DELETE_ARCHIVE']; ?>" value="DeleteCollid" style="width:1.2em;">
```

Comment:
Is the input element above the button here intentional?

Reply:
no this isn't not intentional and its fixed now